packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -401,36 +402,45 @@ export abstract class AbstractMessageManager<
    *
    * @param messageParamsWithId - The params for the personal_sign call to be made after the message is approved.
    * @param messageName - The name of the message
+   * @param resultCallbacks - The callback functions to call when the message is finished.
    * @returns Promise resolving to the raw data of the signature request.
    */
   async waitForFinishStatus(
     messageParamsWithId: AbstractMessageParamsMetamask,
     messageName: string,
+    resultCallbacks: AcceptResultCallbacks | undefined,
```

Comment:
nit: If it's an optional parameter, we probably should write it as:
`resultCallbacks?: AcceptResultCallbacks`

Reply:
Thanks for the flag, fixed 👍